package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@digicatapult/dscp-hyproof-api",
-  "version": "0.0.4",
+  "version": "0.1.0",
   "description": "An OpenAPI API service for DSCP",
   "main": "src/index.ts",
   "scripts": {
```

Comment:
Maybe add / edit 

```
npx @digicatapult/dscp-process-management@latest create -h 127.0.0.1 -p 9944 -u //Alice -f ./processFlows.json
```

because localhost it doesn't work for me .. :(

Reply:
this has been updated.